server/lib/packets/post.go

```diff
@@ -1,7 +1,7 @@
 package packets
 
 import (
-	proto "../../proto"
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)